### ISSUE TITLE
umount: /proc: target is busy

### ISSUE THREAD

**Issue by lguohan** [Azure/sonic-buildimage#411]
https://sonic-jenkins.westus.cloudapp.azure.com/job/common/job/buildimage-baseimage-pr/385/console

```
[INFO] Umount all
+ sudo LANG=C chroot ./fsroot fuser -km /proc
/proc:                6095
+ sudo LANG=C chroot ./fsroot umount /proc
umount: /proc: target is busy
        (In some cases useful info about processes that
         use the device is found by lsof(8) or fuser(1).)
+ rm -f /tmp/tmp.AjVwDYDeYQ
+ sudo umount ./fsroot/proc
umount: /sonic/fsroot/proc: target is busy
        (In some cases useful info about processes that
         use the device is found by lsof(8) or fuser(1).)
+ true
+ true
slave.mk:303: recipe for target 'target/sonic-generic.bin' failed
make: Leaving directory '/sonic'
make: *** [target/sonic-generic.bin] Error 1
Makefile:32: recipe for target 'target/sonic-generic.bin' failed
make: *** [target/sonic-generic.bin] Error 2
Build step 'Execute shell' marked build as failure
Adding one-line test results to commit status...
Setting status of ba493b72e4a0fb0170fc0f4c51488641e8da6658 to FAILURE with url https://sonic-jenkins.westus.cloudapp.azure.com/job/common/job/buildimage-baseimage-pr/385/ and message: ' No test results found.'
Using context: baseimage
Finished: FAILURE
```

**Comment by stcheng** [comment]
https://sonic-jenkins.westus.cloudapp.azure.com/job/common/job/buildimage-baseimage-pr/437/console

**Comment by stcheng** [comment]
https://sonic-jenkins.westus.cloudapp.azure.com/job/broadcom/job/buildimage-brcm-all/177/console

**Comment by lguohan** [comment]
addressed in #801
